### DISCUSSION TITLE
error building DLL - undefined reference to _beginthread

### DISCUSSION THREAD

**Discussion by marvmello** [eclipse-cdt/cdt#868] [Q&A]
I do not have much experience with C,  I'm trying to rebuild a DLL that I found an error (was only a missing else/if in one of the .c files)
When trying to build the project i'm receiveg the error undefined reference to _beginthread, from what I gathered this should be native to WIN

Could anyone guide me?

**Comment by jonahgraham** [comment]
Can you shared what you have tried? Is it an open source project you are trying to rebuild?
